server/pecl/settings.py

```diff
@@ -238,9 +238,17 @@
     "Create Answer Tool": "agent.tools.create_answer_tool.CreateAnswerTool",
 }
 
-AVAILABLE_AGENTS = {
-    "Product Search Agent": "agent.core.agents.product_search_react_agent",
-    "Question Answer Agent": "agent.core.agents.tool_calling_agent",
+AVAILABLE_AGENTS: dict[str, dict[str, str]] = {
+    "Product Search Agent": {
+        "agent": "agent.core.agents.product_search_react_agent.agent.ProductSearchReActAgent",
```

Comment:
What's the use case for making this so granular?

Reply:
Yeah, but I don't think that developers having to understand all the implementation details in order to be able to build a simple agent is a good idea. We need to keep it as simple as possible.